### PR TITLE
Improve SQL Scheduler UI

### PR DESCRIPTION
**Pull request by alexdametto**
## Summary of changes
Just some UI improvements.

## Checklist

- [x] Link to issue this PR refers to: https://github.com/crate/cloud/issues/1608
- [x] Relevant changes are reflected in `CHANGES.md`.
- [x] Added or changed code is covered by tests.
- [x] Required Grand Central APIs are already merged.

### CODE REVIEW

File: src/components/QueryStackTraceModal/QueryStackTraceModal.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        <div>
+          <Button onClick={onClose}>OK</Button>
+        </div>
+      }
+    >
+      <div className="max-h-[400px] overflow-auto flex flex-col gap-4">
+        <SyntaxHighlighter language="sql" title="Failed Query Execution">
+          {query}
+        </SyntaxHighlighter>
+        <SyntaxHighlighter title="Stack Trace">{queryError}</SyntaxHighlighter>
```
Comment (SStorm) [inline src/components/QueryStackTraceModal/QueryStackTraceModal.tsx:42]: Does it figure out what language the snippet is in somehow?.. Wonder how it works.

**Review by SStorm — COMMENTED** [review]
Did not review the code (will leave that to Jon!)

However, two observation regarding the cron pattern:
1. We do not support second notation (i.e. can only have 5 fields, not 6) (`* * * * * *` is invalid)
2. We need to handle API errors where you give an invalid pattern, i.e. Georg found that `*/2 1-25 1,2,3 1-2,3 2-7` is accepted by the UI, but is rejected in the backend. In general, we should handle errors that come back from the API, and display them in the form somehow...

File: src/components/QueryStackTraceModal/QueryStackTraceModal.tsx (hunk 1 above)
Comment (alexdametto) [reply src/components/QueryStackTraceModal/QueryStackTraceModal.tsx:42]: No, I think it's like a "default" style (without any language)

**Comment by alexdametto** [comment]
Thanks @SStorm for the review, gonna look at the problems with the CRON schedule now!

**Review by joncombe — APPROVED** [review]
Code looks great. Could the UI hide the "Invalid CRON schedule" message until the user has typed something in? Also. could that warning by in red, not black?

**Pull request merged**